src/components/NDivider/NDivider.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <div :style="dividerStyle"></div>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Divider",
```

Comment:
@venkaats can you name according to nitrozen name convention
e.g. "nitrozen-divider"
so we can differentiate from other components

Reply:
Okay